lib/mayaUsd/render/vp2RenderDelegate/render_delegate.cpp

```diff
@@ -852,7 +852,12 @@ TfToken HdVP2RenderDelegate::GetMaterialBindingPurpose() const { return HdTokens
 TfTokenVector HdVP2RenderDelegate::GetShaderSourceTypes() const
 {
 #ifdef WANT_MATERIALX_BUILD
-    return { HdVP2Tokens->mtlx, HdVP2Tokens->glslfx };
```

Comment:
We do not yet have a MaterialX solution for Dx11 and Cg so we will not advertise MaterialX support when running under these APIs.